situp-plugins/elasticsearch/README.md

```diff
@@ -14,6 +14,8 @@ pipeline:
   sink:
     elasticsearch:
       hosts: ["http://localhost:9200"]
+      username: admin
+      password: admin
```

Comment:
I'd probably say something like:
```
username: YOUR_USERNAME_HERE
password: YOUR_PASSWORD_HERE
```

Reply:
Also, why this is a good first pass having username/password in plaintext in a config file is dangerous so we might want to have some other methods to pull username/password for the customer in a secure way.

Reply:
Will sync with @kowshikn on this.

Reply:
> Also, why this is a good first pass having username/password in plaintext in a config file is dangerous so we might want to have some other methods to pull username/password for the customer in a secure way.

Notes. This is only for ODFE, not sure if we need to do something extra but will look into it later. In the AES, we will pull it from aws configure.